content/blog/2023/nextflow-kubernetes-enhancements.md

```diff
@@ -0,0 +1,153 @@
+title=Nextflow Kubernetes enhancements
+date=2022-12-20
+type=post
+description=It seems that Kubernetes is everywhere we look! Nextflow users increasingly see K8s as a viable compute environment for their bioinformatic pipelines. In this article, Ben Sherman, Seqera Lab’s master of all things Kubernetes, details recent improvements to the Nextflow/Kubernetes integration.
+image=img/kubernetes-enhancements.jpg
+tags=nextflow, kubernetes
+status=published
+author=Ben Sherman
+icon=ben.jpg
+~~~~~~
+
+Hi, my name is Ben, and I’m a software engineer at Seqera Labs. I joined Seqera in November 2021 after finishing my Ph.D. at Clemson University. I work on a number of things at Seqera, but my primary role is that of a Nextflow core contributor.
```

Comment:
I find the term "Nextflow core" quite jarring 😅  Whilst accurate, is there another term we can use?

Reply:
lol, Nextflow core is not nf-core 🤣

Reply:
I know! But people call nf-core "Nextflow core" a lot, so I've kind of got used to reading it as such 😅

Reply:
any suggestion?

Reply:
Nextflow maintainer?

Reply:
Yeah I usually say Nextflow maintainer, or developer 👍🏻

Reply:
```suggestion
Hi, my name is Ben, and I’m a software engineer at Seqera Labs. I joined Seqera in November 2021 after finishing my Ph.D. at Clemson University. I work on a number of things at Seqera, but my primary role is that of a Nextflow maintainer.
```